modules/Ui.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable  no-restricted-globals */
+/* eslint-disable  no-unused-vars */
+/* eslint-disable  no-undef */
+import ManageBooks from './Book_Management.js';
```

Comment:
- [ ] please avoid turning off the linters in your project, linters are used to optimize your code and make it more readable, therefore we advise you to keep them turned on.